.github/workflows/push.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+on:
+  push:
+    branches: [main]
+jobs:
+  main:
+    name: Deploy to GitHub pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: w3c/spec-prod@v2
+        with:
+          GH_PAGES_BRANCH: gh-pages
+          TOOLCHAIN: bikeshed
+          SOURCE: index.bs
+          DESTINATION: index.html 
```

Comment:
```suggestion
```

Reply:
`TOOLCHAIN`, `SOURCE`, and `DESTINATION` can be omitted. They’re only necessary in the case where w3c/spec-prod can’t infer them from the filename.

The way it works is: w3c/spec-prod looks for any file with a `.bs` extension in the repo root, and if it finds one, it uses that as the source, and infers that the source is a Bikeshed document, and the destination file should have the same base name as the source file (so `index.bs` → `index.html`).